docs/devel/developer-guides/vagrant.md

```diff
@@ -130,225 +150,234 @@ To view the services on any of the nodes:
+list the nodes that you have started with:
 
-```sh
+```shell
 ./cluster/kubectl.sh get nodes
 ```
 
 ### Running containers
 
-Your cluster is running, you can list the nodes in your cluster:
-
-```console
-$ ./cluster/kubectl.sh get nodes
-
-NAME                     LABELS                                          STATUS
-kubernetes-node-0whl     kubernetes.io/hostname=kubernetes-node-0whl     Ready
-kubernetes-node-4jdf     kubernetes.io/hostname=kubernetes-node-4jdf     Ready
-kubernetes-node-epbe     kubernetes.io/hostname=kubernetes-node-epbe     Ready
-```
-
-Now start running some containers!
+You can use `cluster/kube-*.sh` commands to interact with your VM machines.
+Before starting a container there will be no pods, services, or Deployments:
 
-You can now use any of the cluster/kube-*.sh commands to interact with your VM machines.
-Before starting a container there will be no pods, services and replication controllers.
-
-```console
-$ cluster/kubectl.sh get pods
-NAME  READY   STATUS    RESTARTS    AGE
+```shell
+$ ./cluster/kubectl.sh get pods
+NAME        READY     STATUS    RESTARTS   AGE
 
-$ cluster/kubectl.sh get services
-NAME  LABELS   SELECTOR    IP(S)    PORT(S)
+$ ./cluster/kubectl.sh get services
+NAME              CLUSTER_IP       EXTERNAL_IP       PORT(S)       SELECTOR               AGE
 
-$ cluster/kubectl.sh get rc
-CONTROLLER  CONTAINER(S)   IMAGE(S)    SELECTOR    REPLICAS
+$ ./cluster/kubectl.sh get deployments
+CONTROLLER   CONTAINER(S)   IMAGE(S)   SELECTOR   REPLICAS
 ```
 
-Start a container running nginx with a replication controller and three replicas
+To Start a container running nginx with a Deployment and three replicas:
 
-```console
-$ cluster/kubectl.sh run my-nginx --image=nginx --replicas=3 --port=80
-CONTROLLER   CONTAINER(S)   IMAGE(S)   SELECTOR       REPLICAS
-my-nginx     my-nginx       nginx      run=my-nginx   3
+```shell
+$ ./cluster/kubectl.sh run my-nginx --image=nginx --replicas=3 --port=80
 ```
 
-When listing the pods, you will see that three containers have been started and are in Waiting state:
+When listing the pods, you will see that three containers have been started and
+are in Waiting state:
 
-```console
-$ cluster/kubectl.sh get pods
-NAME              READY     STATUS    RESTARTS   AGE
-my-nginx-389da    1/1       Waiting   0          33s
-my-nginx-kqdjk    1/1       Waiting   0          33s
-my-nginx-nyj3x    1/1       Waiting   0          33s
+```shell
+$ ./cluster/kubectl.sh get pods
+NAME                        READY     STATUS              RESTARTS   AGE
+my-nginx-3800858182-4e6pe   0/1       ContainerCreating   0          3s
+my-nginx-3800858182-8ko0s   1/1       Running             0          3s
+my-nginx-3800858182-seu3u   0/1       ContainerCreating   0          3s
 ```
 
-You need to wait for the provisioning to complete, you can monitor the nodes by doing:
+You need to wait for the provisioning to complete. To monitor the nodes:
```

Comment:
I think I would remove this section entirely about looking at images on the node.

Users can watch events if they want `cluster/kubectl.sh get events -w` to see things if they want.